### DISCUSSION TITLE
Philips Hue lights very delayed when turned on

### DISCUSSION THREAD

**Discussion by jeppper** [Koenkk/zigbee2mqtt#9101] [General]
I have had Z2M running very stable for more than a year. I have about 40 Philips Hue lights and until recently they all reacted instantly when turning them on. Some weeks ago they started turning on very delayed (many seconds) and one does not turn on at all any more. I tried to repair that one without any problem but only worked for a very short time.
The behaviour seems very random. A bulb might react instantly one time but next time very delayed. 
I haven´t made any changes to the network but updated many of the bulbs some weeks ago.
The network map looks strange I think but have nothing to compare with.

Has anyone else had the same issue? Could the issue be related to the update?

Info: I am running an old version of Home Assistant 0.103.6 and therefore "stuck" with Z2M version 1.18.1
My coordinator is CC2538 + CC2592

**Comment by Vertikar** [comment]
Could be related to this?
https://github.com/Koenkk/zigbee2mqtt/issues/8663

**Reply by jeppper** [reply]
I don´t think so unfortunately. I haven´t updated any kernel. Only thing that was updated was many of the Philips Hue lights.
My kernel is 4.14.98-v7+
By the way link quality for all the lights are between 20 and 100. It seems low if max is 255, but again I don´t know if this is normal.

**Comment by jeppper** [comment]
This is my map. Does it look okay? 
"light_sovevaerelse" is not connected to the mains at the moment because it gets comletely inresponsive all the time.

![map](https://user-images.githubusercontent.com/26521666/136739806-a5fca730-f171-4c7d-9e05-5bc16bc1d065.png)
?
![map 2](https://user-images.githubusercontent.com/26521666/136739812-0505dbf3-ee81-4a78-bfd4-c6a322518373.png)

**Comment by Anonymenet77** [comment]
Hello,

I have exactly the same thing since I switched to version 1.21.2

HUEs are unstable.
I have to unplug them, I no longer have control over them

Best regards,

**Reply by jeppper** [reply]
Hi
Okay but I haven´t even upgraded to 1.21.2
I am on 1.18.1
Did you update your Philips Hue lights recently?

**Reply by Anonymenet77** [reply]
Yes 1.18.1

**Reply by jeppper** [reply]
You said you switched to 1.21.1
What do you mean by 1.18.1 then?

**Reply by Anonymenet77** [reply]
1.21.2 version zigbee2mqtt

1.18.1 Firmware HUE

**Reply by jeppper** [reply]
Okay I see :-)
Mine are 1.881. and 67.88.1
Did you update not long before this issue? I did... I think.

**Reply by Anonymenet77** [reply]
I updated at the same time so I would not be able to say if it is the HUE firmware which bug or the version 1.21.2 zigbee2mqtt

**Reply by jeppper** [reply]
Okay but it might be the HUE update then since I didn´t update Z2M
I hope something can be done. The behaviour right now is not tolerable.

**Reply by Vertikar** [reply]
Might be worthwhile searching through the issues and see if you can find something similar, otherwise raise a new issue

**Reply by jeppper** [reply]
I will do that

**Comment by jeppper** [comment]
Has anyone in here with a bunch of Hue Lights updated their bulbs lately via OTA and had NO problems afterwards? 
Would be interesting to know

**Comment by ccorderod** [comment]
@jeppper I did. I have over 100 hue lamps. My firmware is 1.88.1 or 67.88.1. Everything is fine. No delays. Your map doesn’t look good imho.

**Comment by ccorderod** [comment]
![image](https://user-images.githubusercontent.com/14359723/136909575-3886c7e0-49c1-4d1f-8b7d-9a9da8edd7c5.png)

**Reply by ccorderod** [reply]
This is the current map for one of my zigbee networks

**Reply by jeppper** [reply]
It looks more as I would expect it to look.
Anyhow I have no clue what to do about mine to make it look more correct.

**Reply by Anonymenet77** [reply]
The concern we encounter does not come from the mapping.

My system was stable before I updated

**Reply by jeppper** [reply]
Just a shame you updated both Z2M and Hue at the same time.
Anyway I still think my map looks strange.

**Comment by jeppper** [comment]
After reading https://github.com/Koenkk/zigbee-OTA/pull/47 I wondered if there could be some sort of issue between the new Hue firmware and older Z2M version but Anonymenet77 is running the newest Z2M version so that should not be a problem I guess.
